Makefile

```diff
@@ -214,7 +214,7 @@ test-unstable: node_modules
 
 .PHONY: test-fullstack
 test-fullstack: node_modules
-	$(MAKE) test-with-database FULLSTACK=1 TIMEOUT_M=30 PROVE_ARGS="$$HARNESS t/full-stack.t t/33-developer_mode.t" RETRY=2
+	$(MAKE) test-with-database FULLSTACK=1 TIMEOUT_M=30 PROVE_ARGS="$$HARNESS t/full-stack.t t/33-developer_mode.t"
```

Comment:
How is that setting the number of retry attempts to 7?

Reply:
Ups :smile: 
![image](https://github.com/user-attachments/assets/736a1166-9ea2-4ae7-9c3c-fb2f87cb37d0)

Updated!